tensorflow/python/framework/test_util.py

```diff
@@ -2682,7 +2682,11 @@ def _GetNdArray(self, a):
       else:
         a = self.evaluate(a)
     if not isinstance(a, np.ndarray):
-      return np.array(a)
+      try:
+        return np.array(a)
+      except NotImplementedError:
+        # numpy 1.20+ could have tripped on a tensor in a nested sequence
+        return np.array([self._GetNdArray(i) for i in a])
```

Comment:
Is this really better?  What if a user calls `np.array` on such a sequence?  I think the idea of removing the bad `__array__` method is superior.

Reply:
IMHO, yes it is better. Calling `np.array` on a sequence with a tensor in it is plain wrong ("You shalt not ...") and a user should get a proper error message.

This merely catches the case where we accidentally did it in test_util and try a different approach.